R/advanced_search_helper_functions.R

```diff
@@ -41,6 +41,16 @@ prepare_elasticsearch_term <- function(filter, filterName) {
   }
 }
 
+is_acceptable_language_filter <- function(language) {
+  if (language %in% c('Chinese', 'English', 'German', 'Italian', 'Japanese',
```

Comment:
if the input `language` is always a single character string the comparison to the vector on the right with `%in%` already returns a boolean, so you could remove the if block